website/docs/guides/orchestration/custom-cicd-pipelines/1-creating-custom-cicd-pipelines.md

```diff
@@ -0,0 +1,431 @@
+---
+title: Creating Custom CI/CD Pipelines
+id: 1-creating-custom-cicd-pipelines
+---
+
+# Creating Custom CI/CD Pipelines
+
+## Background
+
+One of the core tenants of dbt is that analytic code should be version controlled. This provides a ton of benefit to your organization in terms of collaboration, code consistency, stability, and the ability to roll back to a prior version. There’s an additional benefit that is provided with your code hosting platform that is often overlooked or underutilized. Some of you may have experience using dbt Cloud’s [webhook functionality](https://docs.getdbt.com/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration) to run a job when a PR is created. This is a fantastic capability, and meets most use cases for testing your code before merging to production. However, there are circumstances when an organization wants to add a bit to this, like running workflows on every commit (linting), or running workflows after a merge is complete. In this article, we will show you how to setup custom pipelines to lint your project, and trigger a dbt Cloud job via the API.
+
+A note on parlance in this article since each code hosting platform uses different terms for similar concepts. From here on out, anytime you see `pipeline` in this article is interchangeable with `action` (GitHub) and `workflow` (GitLab). Additionally, the terms `pull request` (PR) and `merge request` (MR) are also used interchangeably.  
```

Comment:
Small grammar fix: "anytime you see `pipeline` in this article is interchangeable" -> "anytime you see `pipeline` in this article it is interchangeable"

Alt: "anytime you see `pipeline` in this article is interchangeable" -> "`pipeline` in this article is interchangeable"